types/shared.d.ts

```diff
@@ -161,6 +162,27 @@ export interface LocaleSpecificConfig<ThemeConfig = any> {
   themeConfig?: ThemeConfig
 }
 
+export interface AdditionalConfigEntry<ThemeConfig = any>
+  extends LocaleSpecificConfig<ThemeConfig> {
+  /**
+   * Source of current config entry, only available in development mode
+   */
+  src?: string
```

Comment:
where is this used?

Reply:
```ts
if (mode === 'development') (configExports.config as any).VP_SOURCE = file
```

Might be better to make it something like `__src` or something.

Reply:
Ah, nevermind. Stuff with `_` is stripped. I'll update the types.

Reply:
My first attempt was `__src__` before I realized that 🤣

Reply:
I forgot to remove the type annotation that you pointed out. Please feel free to clean it up!

Reply:
This is removed in the latest commit